src/xo-app/pool/tab-patches.js

```diff
@@ -1,13 +1,160 @@
-import _ from 'intl'
+import ActionButton from 'action-button'
+import Component from 'base-component'
 import React from 'react'
+import SortedTable from 'sorted-table'
+import TabButton from 'tab-button'
+import _ from 'intl'
+import forEach from 'lodash/forEach'
+import isEmpty from 'lodash/isEmpty'
+import map from 'lodash/map'
```

Comment:
This is what happens when using geek tools without the beard...more seriously, this breaks the value of sorting in case of merging.